### ISSUE TITLE
Fix ownership of layers.conf file

### ISSUE THREAD

**Issue by Bejusek** [coi-gov-pl/puppet-jboss#56]
Right now the following code in [internal/module/registerlayer.pp](https://github.com/coi-gov-pl/puppet-jboss/blob/master/manifests/internal/module/registerlayer.pp)  class creates `layers.conf` file as root.

```
exec { "jboss::module::layer::${layer}":
  command => "/bin/awk -F'=' 'BEGIN {ins = 0} /^layers=/ { ins = ins + 1; print \$1=${layer},\$2 } END {if(ins == 0) print \"layers=${layer},base\"}' > ${jboss::home}/modules/layers.conf",
  unless  => "/bin/egrep -e '^layers=.*${layer}.*' ${jboss::home}/modules/layers.conf",
  require => Anchor['jboss::installed'],
  notify  => Service[$jboss::product],
}
```

On systems with umask other than default 0022, this file may become unreadable by jboss::jboss_user, thus **preventing jboss service from starting**.

This command should be executed as `$jboss::jboss_user`

**Comment by Cosaquee** [comment]
@Bejusek can you do this or some other team member should fix this ?

**Comment by Bejusek** [comment]
Sure, I'll fix it.

**Comment by Bejusek** [comment]
Fixed by merging #57
